lib/command/Command.js

```diff
@@ -132,6 +137,17 @@ class Command {
         return !req;
     }
 
+    cooldownCheck(userID) {
+        if(this.usersOnCooldown.has(userID)) {
+            return false;
+        }
+        this.usersOnCooldown.add(userID);
+        setTimeout(() => {
+            this.usersOnCooldown.delete(userID);
+        }, this.cooldown);
+        return true;
```

Comment:
Shouldn't the two lines above only be set on a successful command (or at least after execute) imo it doesn't make sense to set them here during the check.